openeo/rest/job.py

```diff
@@ -20,17 +21,6 @@
 logger = logging.getLogger(__name__)
 
 
-class JobLogEntry:
```

Comment:
never mind, I just noticed you are reusing it for secondary services as well. Makes sense indeed.

Reply:
Because it is not specific to jobs, it is also used in services and potentially sync processing in the future. So I removed the prefix Job and moved it to a neutral place.